integrationtests/git/git_test.go

```diff
@@ -262,8 +264,11 @@ func TestLatestCommitSSH(t *testing.T) {
 			f := git.Fetch{}
 			latestCommit, err := f.LatestCommit(ctx, test.gitjob, client)
 
-			if !reflect.DeepEqual(err, test.expectedErr) {
-				t.Errorf("expecter error is: %v, but got %v", test.expectedErr, err)
+			// works with nil and wrapped errors
+			if test.expectedErr == nil {
+				require.NoError(err)
+			} else {
+				require.Contains(test.expectedErr.Error(), err.Error())
```

Comment:
The scope of the `t` is different from the one of the initialized `require` at the beginning of the test:

```go
require := require.New(t)
// ...
t.Run(name, func(t *testing.T) {
// ...
}
```

I guess this will fail/terminate the whole Test instead of the subtest. Since this is a `require` and not an `assert` it will fail immediately anyway. Do we want to execute the rest, or fail immediately?

Reply:
I'm ok with failing the complete test.